ci/run_black.ps1

```diff
@@ -1,16 +1,17 @@
 $ExitCode = 0
-$GitDiff = git diff --name-only origin/master | Select-String -Pattern ".*\.py" | Select-String -NotMatch ".*/project_template/.*"
+$GitDiff = git diff --name-only origin/master
+$GitDiffBlack = git diff --name-only origin/master | Select-String -Pattern ".*\.py" | Select-String -NotMatch ".*/project_template/.*"
 # If there is no diff between master, then black everything.
 if ( $GitDiff -eq $null ) {
```

Comment:
You should probably be checking against `$GitDiffBlack` here because that is all you care about when deciding whether or not to run black.